### ISSUE TITLE
Switch to Windows -- Docker won't start 

### ISSUE THREAD

**Issue by Amondale** [dotnet/AspNetCore.Docs#25213]
[huh?](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?view=aspnetcore-6.0#run-in-a-windows-container)

Switched to windows in docker, even though I needed to reboot several times before the context menu showed "Switch to Linux" but it appeared the built container ended up being linux, not Win Core (never heard of "ipconfig"):

OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "ipconfig": executable file not found in $PATH: unknown

Also note container_linux.go:380 above. Did I miss a step, or did the docs?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3653642e-61ef-717b-5436-9bd0a3c4e6f4
* Version Independent ID: 80c1ce51-46fb-0693-525a-7cad37150c7b
* Content: [Docker images for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?view=aspnetcore-6.0)
* Content Source: [aspnetcore/host-and-deploy/docker/building-net-docker-images.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/host-and-deploy/docker/building-net-docker-images.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
Hello! Thanks for contacting us. This sounds like a general question about using ASP.NET Core. While we try to look at and respond to all issues, for questions like this we recommend posting to a community support group like [Stack Overflow](https://stackoverflow.com) with the `asp.net-core` tag.

* [Stack Overflow](https://stackoverflow.com/questions)
* [ASP.NET Core Slack](http://tattoocoder.com/aspnet-slack-sign-up/)
* [ASP.NET Gitter](https://gitter.im/aspnet/Home)

Potential bugs in the ASP.NET Core framework and product feedback can be reported at [dotnet/aspnetcore issues](https://github.com/dotnet/aspnetcore/issues) (include full repro steps and cross-link to a repro project in GH when appropriate). For Visual Studio, use the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) gesture or [Suggest a Feature](https://docs.microsoft.com/visualstudio/ide/suggest-a-feature?view=vs-2019) gesture within VS, which open internal issues for the VS team. For Visual Studio Code, you can open an issue at [microsoft/vscode issues](https://github.com/Microsoft/vscode/issues).

**Comment by Amondale** [comment]
Hi Rick! I ran the problem down, and I feel the issue is worth mentioning in the article: 
- Windows 11 Home Edition doesn't support Hyper-V
- Hyper-V is needed by Docker in order to correctly "Switch to Windows Containers"
- Docker fails silently evidently even while displaying "Switch to Linux Containers"

Looking over the Readme.MD file, I saw instructions for creating the Nano container:
## Build an image for Windows Nano Server

The following example demonstrates targeting Windows Nano Server (x64) explicitly (you must have [Windows containers enabled](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers)):

```console
docker build --pull -t aspnetapp:nanoserver -f Dockerfile.nanoserver-x64 .
docker run --rm -it -p 8000:80 aspnetapp:nanoserver
```
And, to my surprise, the image was successfully built as a *Linux* container. 

In short, I think you can save your readers a whole lot of grief by mentioning "don't try this with Windows Home."
